colorbleed/plugins/maya/publish/validate_single_assembly.py

```diff
@@ -3,10 +3,19 @@
 
 
 class ValidateSingleAssembly(pyblish.api.InstancePlugin):
-    """Ensure all nodes are in a single assembly
+    """Ensure the instance is present in the assemblies
```

Comment:
The new docstring here makes _less sense_ to me.

> Ensure the instance is present in the assemblies

This isn't true. The node in the instance are *not required* to be an assembly, they don't need to be a top node. They just need to be _all together_ in a single contained hierarchy.

Could you simplify this docstring in a way that it's simpler?